electron/src/electron/filesystem.ts

```diff
@@ -44,30 +44,35 @@ export class FilesystemPluginElectron extends WebPlugin implements FilesystemPlu
       if(Object.keys(this.fileLocations).indexOf(options.directory) === -1)
         reject(`${options.directory} is currently not supported in the Electron implementation.`);
       let lookupPath = this.fileLocations[options.directory] + options.path;
-      this.NodeFS.readFile(lookupPath, options.encoding, (err:any, data:any) => {
+      this.NodeFS.readFile(lookupPath, options.encoding || 'binary', (err:any, data:any) => {
         if(err) {
           reject(err);
           return;
         }
 
-        resolve({data});
+        resolve({ data: options.encoding ? data : data.toString('base64') });
```

Comment:
write is fine now, but read is returning the converted string, not the original base64 written string.
In example, if I write "e", in the file it's written as "hello", and if I read it, I get "hello" too, but I should get "e" as that's what I wrote. On Android and iOS it works like that.
So I think it should be something like this:
```suggestion
        resolve({ data: options.encoding ? data : Buffer.from(data).toString('base64') });
```

Reply:
It bugs me because `fs.readFile` returns a Buffer to the callback so I shouldn't have wrap it again in a Buffer 🤔

Reply:
Looks like it returns a string if you pass `'binary'` as encoding, but returns a Buffer if you pass `null` as encoding. At least that's what I'm getting.

Reply:
The trick was to add `'binary'` as encoding when using `Buffer.from`. Works now! But I also saw that the `data:application/pdf;base64,` is not stripped on `web`. Should I fix it in this PR? Or should I make another one?

Reply:
better a new one